src/styles/App/_navigation.scss

```diff
@@ -0,0 +1,29 @@
+nav {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 9999;
+  width: 100%;
```

Comment:
shouldn't be 9999, the modal is only 400 on the z-index, and should cover the nav. Can you make it 300?